modules/oe_webtools_cookie_consent/src/Plugin/Filter/FilterIframeCck.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_cookie_consent\Plugin\Filter;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Drupal\filter\FilterProcessResult;
+use Drupal\filter\Plugin\FilterBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a filter to prepend Cookie Consent Kit on iframe elements.
+ *
+ * @Filter(
+ *   id = "filter_iframe_cck",
+ *   title = @Translation("Apply cookie consent to iframes"),
+ *   description = @Translation("Alters the src attribute of iframes to add the Webtools Cookie Consent Kit."),
```

Comment:
```suggestion
 *   description = @Translation("Alters the <code>src</code> attribute of iframes to add the Webtools Cookie Consent Kit."),
```
That's what I meant with monospace.